tools/anthosbm-ansible-module/roles/anthos/templates/cluster.yaml.j2

```diff
@@ -0,0 +1,101 @@
+{#
```

Comment:
Please remove the open and closed brackets form the license header

Reply:
The comments in jinja templates are created by enclosing the text between {# and #}. So the curly bracket is required.